src/Sentiment/index.js

```diff
@@ -0,0 +1,95 @@
+import * as tf from '@tensorflow/tfjs';
+import callCallback from '../utils/callcallback';
+/**
+* Initializes the Sentiment demo.
+*/
+
+const OOV_CHAR = 2;
+const PAD_CHAR = 0;
+
+function padSequences(sequences, maxLen, padding = 'pre', truncating = 'pre', value = PAD_CHAR) {
+  return sequences.map((seq) => {
+    // Perform truncation.
+    if (seq.length > maxLen) {
+      if (truncating === 'pre') {
+        seq.splice(0, seq.length - maxLen);
+      } else {
+        seq.splice(maxLen, seq.length - maxLen);
+      }
+    }
+    // Perform padding.
+    if (seq.length < maxLen) {
+      const pad = [];
+      for (let i = 0; i < maxLen - seq.length; i += 1) {
+        pad.push(value);
+      }
+      if (padding === 'pre') {
+        // eslint-disable-next-line no-param-reassign
+        seq = pad.concat(seq);
+      } else {
+        // eslint-disable-next-line no-param-reassign
+        seq = seq.concat(pad);
+      }
+    }
+    return seq;
+  });
+}
+
+class Sentiment {
+  constructor(callback) {
```

Comment:
I wonder if it might make sense to add a `String` as argument indicating the name of the model. This would be similar to how we let the user choose between `MobileNet` and `Darknet` with image classification. Even if we just have one model supported right now, it is also important pedagogically so that a user of ml5 is reminded there is a pre-trained model, based on a dataset from someone else!

Reply:
fixed this, adding to next pull request!
(: